telemetry/definitions/commonDefinitions.json

```
@@ -1184,6 +1184,11 @@
             "type": "int",
             "description": "Number of references in response"
         },
+        {
+            "name": "cwsprChatRequestIds",
+            "type": "string",
+            "description": "The list of request ids corresponding to this user message. Request ids must be in chronological order"
+        },
```

No need for this to be a chat-specific field. Other metrics may want to have a list of requestIds. https://github.com/aws/aws-toolkit-common/blob/576bf05151406c1b35f96ab6923576785dab1a63/telemetry/definitions/commonDefinitions.json#L1945-L1954
```suggestion
            "name": "requestIds",
            "type": "string",
            "description": "Request ids associated with an task or action, in chronological order."
        },
```

---

Resolved